packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Snippets/TypedSnippetsTests.cs

```diff
@@ -8,6 +8,7 @@
 using System.IO;
 using System.Reflection;
 using System.Text.Json;
+using Microsoft.Generator.CSharp.ClientModel.Primitives;
```

Comment:
nit: remove

Reply:
nevermind

Reply:
needed because ScmKnownParameters moved to this namespace since that is the convention with MGC